src/ray/raylet/scheduling_queue.cc

```diff
@@ -6,49 +6,90 @@ namespace ray {
 
 namespace raylet {
 
+bool SchedulingQueue::TaskQueue::Append(const TaskID &task_id, const Task &task) {
```

Comment:
general style comment: we try to make all variable names use full words (concatenated with an _ ) and, to the extent possible, we strive to make those variable names as descriptive as possible. Examples `list_iter`, `list_`, `map_`

Reply:
Got it. Thanks for the comment!:)